pkg/ssh/manager_test.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/gliderlabs/ssh"
 	"github.com/okteto/okteto/pkg/log"
```

Comment:
pending log statements to be refactored

Reply:
done at https://github.com/okteto/okteto/pull/2090/commits/e94628c8dfce42213346c67e745b2bc3e0d235fe